src/test/java/org/folio/inventory/validation/experimental/TargetItemStatusValidatorsTest.java

```diff
@@ -0,0 +1,95 @@
+package org.folio.inventory.validation.experimental;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import lombok.SneakyThrows;
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import org.folio.inventory.domain.items.Status;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.inventory.domain.items.ItemStatusName.IN_PROCESS;
+import static org.folio.inventory.domain.items.ItemStatusName.forName;
+
+@RunWith(JUnitParamsRunner.class)
+public class TargetItemStatusValidatorsTest {
```

Comment:
There is a new class specifically for the `In process` target status. Is this class still needed?

Reply:
This comment was marked as resolved, how has it been resolved?